app/src/main/java/com/rizadwi/snapsift/util/Clipper.kt

```diff
@@ -0,0 +1,18 @@
+package com.rizadwi.snapsift.util
+
+import javax.inject.Inject
+
+class Clipper @Inject constructor() {
```

Comment:
is that really necessaru to use inject here?

Reply:
it is, i need the Clipper utility in the [ArticleAdapter](https://github.com/rizadwiandhika/snap-sift/blob/d9967f39c7a142e78867a8b679220a85359ffe4f/app/src/main/java/com/rizadwi/snapsift/presentation/adapter/ArticleAdapter.kt#L19)